### ISSUE TITLE
[Future]: Auto-Generated Create and Update Dates

### ISSUE THREAD

**Issue by AgedLace** [mkusz/mkdocs-publisher#53]
### Future description

```markdown
**Future description**

Found a process that will create `date` and `update` yaml keys only in files that contain these yaml keys.  

If you don't want the `date` and `update` keys in certain documentation files, omit these two yaml keys.  

For example, to only have these metadata keys in blog-posts just add them into the blog-post template file.  Don't add these two keys to any other templates or files.

The process is documented in the 'source/blog/Test Auto Update.md' file in the repo referred to below.
```

### Project plugin

- [ ] pub-blog
- [ ] pub-debugger
- [ ] pub-meta
- [ ] pub-minifier
- [X] pub-obsidian
- [ ] pub-social
- [ ] a new plugin
- [ ] external plugin integration

### Link

https://github.com/AgedLace/Auto-Generate-Dates

### Contact details

agedlace@yandex.com

**Comment by mkusz** [comment]
You are right, that this should be optional, but I need to extend documentation with information on how it will affect certain SEO. MkDocs by default, when generated a `sitemap.xml` file, uses the date of the generation of the document. Meta plugin changes it by the date degined in `update` value (if this value is not existing, `date` of creation is used). The best approach (I need to add this to the documentation) is by using Obsidian linter plugin, that can handle those values automatically.

**Comment by AgedLace** [comment]
I agree that Linter does this best. However, I could not figure out how to limit which files received the creation and update dates.  It placed them into every file.

**Comment by mkusz** [comment]
Take a look at the below screen:
![image](https://github.com/mkusz/mkdocs-publisher/assets/9740637/9b7a875d-aa41-4f98-a082-5122e84ceefd)
